### PR TITLE
Fixed Sphinx warning errors

### PR DESCRIPTION
**Pull request by ron-unstructured**
Fixed issue #1437 - resolved the Warning errors when building sphinx with `make html`.

test:
1. `cd docs` folder and `rm -rf build`
2. `pip install -r requirements.txt`
3. run `make html`

### CODE REVIEW

File: docs/requirements.txt [hunk 1]
```diff
@@ -41,6 +41,7 @@ jinja2==3.1.2
     # via sphinx
 markupsafe==2.1.3
     # via jinja2
+myst-parser==2.0.0
```
Comment (cragwolfe) [inline docs/requirements.txt:44]: would this get lost the next time someone does `make pip-compile` due to:
 https://github.com/Unstructured-IO/unstructured/blob/a5ca628/Makefile#L228 ?

File: requirements/build.in [hunk 2]
```diff
@@ -20,3 +20,6 @@ furo==2023.7.26
 
 # NOTE(robinson) - The following dependencies are pinned to address security scans
 certifi>=2022.12.07
+
+# NOTE(ronny) - Added to suppress Sphinx warnings
+myst-parser==2.0.0
```
Comment (qued) [inline requirements/build.in:25]: Is it necessary to have the pin, i.e. `myst-parser==2.0.0` as opposed to `myst-parser`?

**Review by qued — COMMENTED** [review]
Overall looks good, would prefer to unpin `myst-parser` unless there's a good reason for the strict pin.

**Review by qued — APPROVED** [review]
LGTM!

**Pull request merged**